src/test/java/org/hbase/async/HBaseClientIT.java

```diff
@@ -80,9 +81,61 @@ public static void deleteTable() throws IOException {
     }
   }
 
-  private static HBaseClient client;
+  public void testEnsureTableFamilyExists() throws Exception {
+    client.ensureTableFamilyExists(TABLE_NAME.toBytes(), FAMILY).join();
+    Assert.assertTrue(true);
+  }
+
+  @Test(expected=NoSuchColumnFamilyException.class)
+  public void testEnsureTableFamilyExists_nocf() throws Exception {
+    client.ensureTableFamilyExists(TABLE_NAME.toBytes(), Bytes.toBytes("nonExistingCF")).join();
+  }
+  
+  @Test(expected=TableNotFoundException.class)
+  public void testEnsureTableFamilyExists_noTable() throws Exception {
+    client.ensureTableFamilyExists("nonExistingTable".getBytes(), Bytes.toBytes("nonexistingCF")).join();
+  }
+
+  @Test
+  public void atomicIncrement() throws Exception {
+    byte[] rowKey = dataHelper.randomData("putKey-");
+    byte[] qualifier = Bytes.toBytes("qual");
+    byte[] value = Longs.toByteArray(5);
+
+    client.put(new PutRequest(TABLE_NAME.getName(), rowKey, FAMILY, qualifier, value)).join();
```

Comment:
Please don't join here, but rather do it after `flush().join()`, since this may not be sent until `flush()` is complete.